app/markdown_parser.py

```diff
@@ -24,6 +24,7 @@ def parse_caption_markdown(markdown):
     parsed_text = re.sub(r"_(.+)_", lambda x: "<i>" + x.group(1) + "</i>", parsed_text)
 
     # Replace urls
-    parsed_text = re.sub(r"\[(.+?)\]\((.+?)\)", lambda x: "<a target='_blank' rel='noopener' href=\"" + x.group(2) + "\">" + x.group(1) + "</a>", parsed_text)
+    parsed_text = re.sub(r"\[(.+?)\]\+\((.+?)\)", lambda x: f"<a target='_blank' rel='noopener' href=\"" + x.group(2) + "\">" + x.group(1) + "</a>", parsed_text)
```

Comment:
So whatcha think about doing `[text]+(url)` to open in a new tab?
and `[text](url)` to open in current window?

Reply:
I'm not undown